fastlane/Jetpack-Fastfile

```diff
@@ -159,11 +159,20 @@ lane :download_jetpack_localized_app_store_metadata do
   sh './download_metadata.swift jetpack'
 
   jetpack_metadata_glob = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'jetpack_metadata/**/*.txt')
-  sh "git add #{jetpack_metadata_glob}"
-  git_commit(
-    path: jetpack_metadata_glob,
-    message: 'Update Jetpack metadata translations'
-  )
+
+  no_files_changed = sh "git status #{jetpack_metadata_glob}" do |_, output, _|
+    output.include? 'nothing to commit'
+  end
+
+  if no_files_changed
+    UI.message "No translations downloaded into #{jetpack_metadata_glob}. Moving on."
+  else
+    sh "git add #{jetpack_metadata_glob}"
+    git_commit(
+      path: jetpack_metadata_glob,
+      message: 'Update Jetpack metadata translations'
+    )
+  end
```

Comment:
If you find there's something wrong or that needs changing in this code, I'll address is in a followup PR against the release branch so that we don't block the merging back into `develop`. Thanks 😄 .